CHANGELOG.md

```diff
@@ -1,17 +1,6 @@
-## <small>5.0.4 (2021-02-21)</small>
-
-* [Config Change]Button route url update (#8) ([99ac4ec](http://github.paypal.com/paypal/paypal-auth/commit/99ac4ec)), closes [#8](http://github.paypal.com/paypal/paypal-auth/issues/8)
-* Add button labels for sq_AL, si_LK (#13) ([c23d057](http://github.paypal.com/paypal/paypal-auth/commit/c23d057)), closes [#13](http://github.paypal.com/paypal/paypal-auth/issues/13)
-* Add CODEOWNERS ([030aea7](http://github.paypal.com/paypal/paypal-auth/commit/030aea7))
-* Add new translations bg, et, lt, lv, ms, sl, tl (#10) ([f31e6da](http://github.paypal.com/paypal/paypal-auth/commit/f31e6da)), closes [#10](http://github.paypal.com/paypal/paypal-auth/issues/10)
-* Add Vietnamese (vi) and Romanian (ro) (#3) ([2b4064f](http://github.paypal.com/paypal/paypal-auth/commit/2b4064f)), closes [#3](http://github.paypal.com/paypal/paypal-auth/issues/3)
-* Alphabetical sort of L10n content blocks (#4) ([df8ff8a](http://github.paypal.com/paypal/paypal-auth/commit/df8ff8a)), closes [#4](http://github.paypal.com/paypal/paypal-auth/issues/4)
-* comment out test task before publish, flow's broken right now and I dont know why ([f39011e](http://github.paypal.com/paypal/paypal-auth/commit/f39011e))
-* Pin to specific flow-bin version ([32b9f40](http://github.paypal.com/paypal/paypal-auth/commit/32b9f40))
-* Pin to specific flow-bin version ([dcdcaf6](http://github.paypal.com/paypal/paypal-auth/commit/dcdcaf6))
-* Update CODEOWNERS ([0579719](http://github.paypal.com/paypal/paypal-auth/commit/0579719))
-* Update README.md ([3b38104](http://github.paypal.com/paypal/paypal-auth/commit/3b38104))
-* Upgrade .travis.yml to always follow latest Node LTS (Node 12) (#6) ([6fcd736](http://github.paypal.com/paypal/paypal-auth/commit/6fcd736)), closes [#6](http://github.paypal.com/paypal/paypal-auth/issues/6)
+## <small>5.0.4-alpha5.1 (2021-10-04)</small>
```

Comment:
Please back out this changelog change. There should be no changes to CHANGELOG.md

Reply:
done